templates/definition/meter/fox-ess-h1-ethernet.yaml

```diff
@@ -1,8 +1,9 @@
-template: fox-ess-ethernet
+template: fox-ess-h1
+covers: [fox-ess-ethernet]
```

Comment:
Der Form halber:
```suggestion
covers: ["fox-ess-ethernet"]
```

Reply:
Verwende für den FOXESS H3 diese Register. Das passt. Leider kann man die H3 aufgrund von Hersteller Restriktionen nicht per Ethernet anbinden. Geht nur per RS485. Zusätzlich kann man noch den mitgelieferten Meter auslesen. Interesse an den zugehörigen Registern? Dann könnte man das Ganze in ein Template für den H3 und Modus gießen...

Reply:
Gerne- schau dir aber bitte vorher mal das neue H3 template an!

Reply:
Habe ich mir gerade mal angeschaut. Sieht passend aus. Außer, dass man mit den FOXESS nicht mehr über Ethernet kommunizieren kann (herstellerseitig gesperrt). Es würde also eine "fox-ess-H3-rs485.yaml" Sinn machen, da man nur noch über die serielle Schnittstelle an die Daten kommt. Oder habe ich da einen Denkfehler? Die Register für den mitgelieferten CHINT Meters sende ich Dir noch zu. Damit hätte man dann auch den Hausverbrauch mit im System.

Reply:
Korrekt. Im Template müsste es hier:

```
  - name: modbus
    choice: ["tcpip"]
```

`rs485` heissen was dann natürlich auch impliziert, dass ein Modbus Adapter verwendet (und konfiguriert) werden kann. Die ganze Diskussion sollte jetzt zu einem Folge-PR führen ;)